components/account/AccountLiveDataCard.tsx

```diff
@@ -1,70 +1,139 @@
+import moment from "moment";
+interface AccountLiveDataProps {
+  accountName: string;
+  liveDataOperations: boolean;
+  setLiveDataOperations: (state: boolean) => void;
+  refetchAccountOperations: any;
 }
 
-const AccountLiveDataCard: React.FC<CardHeaderProps> = ({ blockDetails }) => {
-  const { settings, setSettings } = useUserSettingsContext();
-  const {
-    explorerBlockNumber,
-    hiveBlockNumber,
-    loading: isLoading,
-  } = useBlockchainSyncInfo(settings.liveData, 20000); // Pass settings.liveData and the interval
-
-  const blockDifference = getBlockDifference(
-    hiveBlockNumber,
-    explorerBlockNumber
+const AccountLiveData: React.FC<AccountLiveDataProps> = ({ 
+    accountName, 
+    liveDataOperations,
+    setLiveDataOperations,
+    refetchAccountOperations }) => {
+  const [liveDataManabars, setLiveDataManabars] = useState(false);
+  const { manabarsData, refetchManabars } = useManabars(accountName);
+  const [liveDataDetails, setLiveDataDetails] = useState(false);
+  const {accountDetails, isAccountDetailsLoading, isAccountDetailsError, notFound, refetchAccountDetails} = useAccountDetails(accountName);
+  const [liveDataAuth, setLiveDataAuth] = useState(false);
+  const {accountAuthoritiesData, accountAuthoritiesDataLoading, accountAuthoritiesDataError, refetchAccountAuthorities} = useAccountAuthorities(accountName);
+  const [liveDataRc, setLiveDataRc] =  useState(false);
+  const {rcDelegationsData, isRcDelegationsLoading, isRcDelegationsError, refetchRcDelegations} = useRcDelegations(accountName, 1000);
+  const [liveDataVesting, setLiveDataVesting] = useState(false);
+  const {vestingDelegationsData, isVestingDelegationsLoading, isVestingDelegationsError, refetchVestingDelegations} = useVestingDelegations(accountName, null, 1000);
+  const [liveDataWitnessVoters, setLiveDataWitnessVoters] = useState(false);
+  const {witnessVoters, isWitnessVotersLoading, isWitnessVotersError, refetchWitnessVoters,} = useWitnessVoters(accountName, false, true, "vests");
+  const [liveDataWitnessVotesHistory, setLiveDataWitnessVotesHistory] = useState(false);
+  const [fromDate, setFromDate] = useState<Date>(
+    moment().subtract(7, "days").toDate()
   );
-
-  const isLiveDataToggleDisabled =
-    blockDifference > config.liveblockSecurityDifference || isLoading;
-
+  const [toDate, setToDate] = useState<Date>(moment().toDate());
+  const {votesHistory, isVotesHistoryLoading, isVotesHistoryError, refetchVotesHistory} = useWitnessVotesHistory(accountName, false, fromDate, toDate);
   useEffect(() => {
-    const interval = setInterval(() => {
-      if (settings.liveData) {
-        // Trigger the refetch here
-        setSettings({ ...settings });
-      }
-    }, 20000);
-
-    return () => clearInterval(interval);
-  }, [settings, setSettings]);
-
+    const intervals: NodeJS.Timeout[] = [];
+  
+    if (liveDataManabars) {
+      intervals.push(setInterval(() => {
```

Comment:
instead of defining multiple intervals, define a single one, and inside you can do the checks (if conditions)